src/transaction/log_recovery.c

```diff
@@ -1753,6 +1767,19 @@ log_rv_analysis_assigned_mvccid (THREAD_ENTRY * thread_p, int tran_id, LOG_LSA *
   auto rec = (const LOG_REC_ASSIGNED_MVCCID *) (log_page_p->area + log_lsa->offset);
   tdes->mvccinfo.id = rec->mvccid;
 
+  if (is_passive_transaction_server ())
```

Comment:
Nit.
Actually, the change in `log_recovery_context` could be done regardless of the type of transaction server.
On an ATS the result will simply not be used.

Reply:
I agree with you. However, I would like to leave this code as is, so that readers can more easily understand the context of this code.